packages/web/src/scss/components/Icon/_Icon.scss

```diff
@@ -0,0 +1,18 @@
+@use '@tokens' as tokens;
+@use '../../tools/responsive-properties';
+@use 'theme';
+
+svg {
```

Comment:
oh, I would be more careful with this. I think the solution should be a new (optional) `.Icon`.